README.md

```diff
@@ -22,12 +22,17 @@ $ brew install <formula>
 
 For example:
 
- * Install the latest available production release of MongoDB Community Server ([including all command line tools](https://docs.mongodb.com/manual/reference/program/)). This will currently install MongoDB 4.2.x:
+ * Install the latest available production release of MongoDB Community Server ([including all command line tools](https://docs.mongodb.com/manual/reference/program/)). This will currently install MongoDB 4.4.x:
    ```
    $ brew install mongodb-community
    ```
 
- * Install the latest 4.2.x production release of MongoDB Community Server and command line tools:
+ * Install the latest 4.e.x production release of MongoDB Community Server and command line tools:
+   ```
+   $ brew install mongodb-community@4.4
```

Comment:
I didn't see the versioned install script for this anywhere. Did I miss something? It's been a while since I interacted with Homebrew much, but I was sure every versioned install needed its own file, and you cold redirect to it from the main script if you wanted.

Reply:
Yeah this threw me at first too.

It appears that the way it is supposed to work is that the "latest" should be in the unversioned file, and then in the `Aliases` folder you link to that unversioned one under a versioned name. It felt odd to me. Shouldn't it go the other way? With the `Formula` directory containing only the versioned ones, and then the `Aliases` directory containing an unversioned link? Anyway, I poked around and looked at some other homebrew projects, and they all seem to do it the way we are doing it here. Maybe it is all just copypasta, or maybe there is a good reason. But I don't think we should change it now.

It is a little hard to see in the the diff, but the very first change is a symlink redirect:

` Aliases/mongodb-community@4.2 → Aliases/mongodb-community@4.4 `

That is the relevant update.

Reply:
I guess it makes sense... I just had fears that someone would not be able to install other versions if they mistakenly installed the unversioned one. Maybe those fears are unfounded, though.